tutorials/spanruler_restaurant_reviews/patterns.jsonl

```diff
@@ -0,0 +1,35 @@
+{"label": "Rating", "pattern": [{"IS_DIGIT": true}, {"LOWER": "star"}, {"LOWER": "review", "OP": "?"}]}
```

Comment:
For this I don't mind this approach either. I used the `patterns.jsonl` file because it's the most apparent one in [the documentation](https://spacy.io/usage/rule-based-matching/#spanruler-patterns). Using a registry makes sense too, but I haven't seen this one documented somewhere (I may be wrong) and it might cause a confusion?

Reply:
hmm true its not documented. I prefer it due to the versioning, wonder what others think also about this pattern vs. the `jsonl`. I think for packaging it also makes more sense to have the rules in the code?

Reply:
Yeah I like the idea of versioning rules too! Makes for a great "best practice" thing if possible. We can also update the docs if everyone agrees.

Reply:
Yes, I definitely think versioning the rules is the "spaCy v3 way" to go - good catch Ákos!